modules/catalog/src/main/scala/lucuma/catalog/telluric/TelluricStar.scala

```diff
@@ -27,6 +34,22 @@ case class TelluricStar(
 ):
   val simbadName: NonEmptyString = NonEmptyString.unsafeFrom(s"HIP $hip")
 
+  def asSiderealTarget: Target.Sidereal =
+    Target.Sidereal(
+      name = simbadName,
+      tracking = SiderealTracking(
+        baseCoordinates = coordinates,
+        epoch = Epoch.J2000,
+        properMotion = None,
+        radialVelocity = None,
+        parallax = None
+      ),
+      sourceProfile = SourceProfile.Point(
+        SpectralDefinition.BandNormalized(None, SortedMap.empty)
+      ),
+      catalogInfo = None
```

Comment:
I guess we know it is Simbad but don't have the other CatalogInfo info?

Reply:
I suppose we could add a TelluricServer catalog info